selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -99,15 +99,25 @@ void WifiManager::refreshFinished(QDBusPendingCallWatcher *watcher) {
     auto properties = replay.value();
 
     const QByteArray ssid = properties["Ssid"].toByteArray();
-    uint32_t strength = properties["Strength"].toUInt();
-    if (ssid.isEmpty() || (seenNetworks.contains(ssid) && strength <= seenNetworks[ssid].strength)) continue;
+    if (ssid.isEmpty()) {
+      continue;
+    }
+
+    // Add entry for each seen SSID
+    if (!seenNetworks.contains(ssid)) {
+      seenNetworks[ssid] = {};
+    }
 
-    SecurityType security = getSecurityType(properties);
-    ConnectedType ctype = ConnectedType::DISCONNECTED;
+    seenNetworks[ssid].security_type = getSecurityType(properties);
     if (path.path() == activeAp) {
```

Comment:
@deanlee look like this is only true for the current access point, right? If so, does my change LGTY?

Reply:
lgtm